networkx/algorithms/bipartite/tests/test_matching.py

```diff
@@ -33,188 +26,95 @@ def setup_method(self):
         self.graph.add_nodes_from(range(12))
         self.graph.add_edges_from(edges)
 
-        # Example bipartite graph from issue 2127
```

Comment:
Changed out complicated specific test case that is disconnected with a separate one that is more simple.